### DISCUSSION TITLE
Python api routes don't work locally (vercel CLI permission error?)

### DISCUSSION THREAD

**Discussion by holdenmatt** [vercel/vercel#9977] [Help]
### Problem Description

I'm trying to add a Python /api function, as described here:
https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/python

- I created a brand new app with `yarn create next-app` and all defaults (but no `app` folder).
- I deleted the `api/` folder from `pages/` and added a single `api/index.py` route in the root of my project.

When I deploy to Vercel, this works.

But when I run `vercel dev` locally and hit the api route in my browser, it crashes with this error:
```
> Building @vercel/python@latest:api/index.py
Installing required dependencies...
> Built @vercel/python@latest:api/index.py [531ms]
Error: spawn /Users/matt/Library/Caches/com.vercel.fun/runtimes/python3/bootstrap EACCES
    at Process.ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:476:16)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
Error: An unexpected error occurred!
Error: spawn /Users/matt/Library/Caches/com.vercel.fun/runtimes/python3/bootstrap EACCES
    at Process.ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:476:16)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
```

I installed the Vercel CLI using:
```
yarn global add vercel
```

Looks like some kind of permission error - how can I fix this?

**Comment by Zakinator123** [comment]
`chmod +x /Users/matt/Library/Caches/com.vercel.fun/runtimes/python3/bootstrap`
